### DISCUSSION TITLE
새로생긴 오류메시지 NECKLACE_211113_2(Clone)

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#120] [Issue Report (Legacy)]
이번에 새로 보이는 오류메시지입니다.
실행은 잘 되요 :0

<img width="590" alt="스크린샷 2021-12-22 오후 8 26 19" src="https://user-images.githubusercontent.com/90679926/147085870-a14efdc0-09e6-4dc6-a8db-082bfd5fcc52.png">

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 오류가 최신 버전에서도 지속적으로 나타나는지 확인 부탁드립니다.

**Reply by x4cf** [reply]
현재는 같은오류가 발견되지 않아요 :D

**Reply by Zepeto-TechEvangelist** [reply]
넵 확인 감사합니다.
